src/lib/exampleKcl.ts

```diff
@@ -5,7 +5,7 @@ const sigmaAllow = 35000 // psi
 const width = 6 // inch
 const p = 300 // Force on shelf - lbs
 const distance = 12 // inches
-const M = 12 * 300 / 2 // Moment experienced at fixed end of bracket
+const M = distance * p / 2 // Moment experienced at fixed end of bracket
 const FOS = 2 // Factor of safety of 2
 const shelfMountL = 8 // The length of the bracket holding up the shelf is 6 inches
```

Comment:
I stared at this more and now I'm realizing now that distance isn't related at all to shelfMountL or wallMountL. Just thinking through the problem, it would make sense to me the moment experienced at the end of the bracket would be related to those and not necessarily distance? 
Also, why do we divide moment by 2? Is this just because we were originally assuming the distance of each arm of the bracket was 1/2 of the original distance of 12?

So, maybe it should be `const M = shelfMountL * p`?

Reply:
Looking back at this, I don't like some of the variable names I chose. 

We are dividing by two because there are two brackets holding up the shelf (assumed at equal distance apart from the load). 
`distance` in this example is supposed to be the shelf length from the wall. I'm going to add this picture to the onboarding to make it more clear and update the code to follow. This is what it should be:

<img width="775" alt="Screenshot 2024-04-29 at 8 03 00 AM" src="https://github.com/KittyCAD/modeling-app/assets/114548659/bddca9c1-ea9e-4d5c-ba9b-91ea1ef77787">

```js
const sigmaAllow = 35000 // psi
const width = 6 // width of the bracket - in
const p = 300 // Force on shelf - lbs
const L = 12 // Length of the shelf from the wall - in
const M = L * p / 2 // Moment experienced at fixed end of bracket. Divided by 2 since there are 2 brackets holding up the shelf.
const FOS = 2 // Factor of safety
const shelfMountL = 8 // The length of the bracket holding up the shelf - in
const wallMountL = 6 // the length of the bracket mounted on the wall - in

// Calculate the thickness off the allowable bending stress and factor of safety
const t = sqrt(6 * M * FOS / (width * sigmaAllow))
```
Calculating the allowable bending stress, we have
$$\sigma = \frac{M*c}{I} FOS$$

where

$$M = \frac{p  L}{2}$$

$$c = \frac{t}{2}$$

$$I = \frac{1}{12} w t^3$$

plugging those in and simplifying:

$$\sigma = \frac{6 F L}{w t^2}$$

so

$$ t = \sqrt{\frac{6 p L FOS}{w \sigma}} $$

I'm going to update the onboarding code and make it clearer. Thanks for bringing this up @iterion!